app/models/person.rb

```diff
@@ -59,7 +59,7 @@ def next_target
   end
 
   def unconvinced_legislators
-    legislators && legislators.unconvinced
+    legislators && legislators.unconvinced.eligible
   end
```

Comment:
yeah, sounds like a good thing to investigate. I'm eager to learn as much as I can about performance optimization -- not something I had to worry about on my last couple projects